samples/openthread/ncp/prj.conf

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+# ===============================
+# KERNEL OPTIONS
+# ===============================
+CONFIG_NEWLIB_LIBC=y
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_INIT_STACKS=y
+# ===============================
+
+# ===============================
+# LOGGING OPTIONS
+# ===============================
+CONFIG_NET_LOG=y
+CONFIG_LOG=y
+CONFIG_NET_STATISTICS=y
+CONFIG_PRINTK=y
+CONFIG_OT_NCP_LOG_LEVEL_DBG=y
+CONFIG_NET_CONFIG_LOG_LEVEL_OFF=y
+
+CONFIG_LOG_BACKEND_RTT=n
+CONFIG_LOG_BACKEND_UART=y
+CONFIG_BOOT_BANNER=n
+# ===============================
+
+# ===============================
+# NETWORKING OPTIONS
+# ===============================
+# ====================
+# OPENTHREAD OPTIONS
+# ====================
+# set OpenThread NCP architecture
+CONFIG_OPENTHREAD_NCP=y
+
+# enable OpenThread
+CONFIG_NET_L2_OPENTHREAD=y
+
+CONFIG_OPENTHREAD_DEBUG=y
+CONFIG_OPENTHREAD_L2_DEBUG=y
+CONFIG_OPENTHREAD_L2_LOG_LEVEL_INF=y
+CONFIG_OPENTHREAD_CHANNEL=11
```

Comment:
I don't think you need to set channel and panid for NCP.

Reply:
Done